test/functional/sqlsrv/srv_230_sqlsrv_buffered_numeric_types.phpt

```diff
@@ -3,51 +3,58 @@ Read numeric types from SQLSRV with buffered query.
 --DESCRIPTION--
 Test numeric conversion (number to string, string to number) functionality for buffered queries with SQLSRV.
 --SKIPIF--
+<?php require('skipif_versions_old.inc'); ?>
 --FILE--
 <?php
 
-require_once("MsCommon.inc");
-
-$conn = connect(array("CharacterSet"=>"utf-8"));
-if (!$conn) {
-    printErrors("Connection could not be established.\n");
+function getInputData($inputs)
+{
+    return array('a' => $inputs[0],
+                 'neg_a'=> $inputs[1], 
+                 'b' => $inputs[2], 
+                 'neg_b' => $inputs[3], 
+                 'c' => $inputs[4], 
+                 'neg_c' => $inputs[5], 
+                 'zero' => $inputs[6], 
+                 'zerof' => $inputs[7], 
+                 'zerod' => $inputs[8]);
 }
 
+require_once('MsCommon.inc');
+
+$conn = AE\connect(array("CharacterSet"=>"utf-8"));
+$tableName = 'test230';
+
 $sample = 1234567890.1234;
 $sample1 = -1234567890.1234;
 $sample2 = 1;
 $sample3 = -1;
 $sample4 = 0.5;
 $sample5 = -0.55;
 
-$query = 'CREATE TABLE #TESTTABLE (a float(53), neg_a float(53), b int, neg_b int, c decimal(16, 6), neg_c decimal(16, 6), zero int, zerof float(53), zerod decimal(16,6))';
-
 // Create table
-$stmt = sqlsrv_query($conn, $query);
-if ($stmt === false) {
-    die(print_r(sqlsrv_errors(), true));
-}
-
-$query = 'INSERT INTO #TESTTABLE (a, neg_a, b, neg_b, c, neg_c, zero, zerof, zerod) VALUES(?, ?, ?, ?, ?, ?, 0, 0, 0)';
-$params = array($sample, $sample1, $sample2, $sample3, $sample4, $sample5);
-
-$stmt = sqlsrv_query($conn, $query, $params);
-if ($stmt === false) {
-    die(print_r(sqlsrv_errors(), true));
-}
-$params = array($sample4, $sample5, 100000, -1234567, $sample, $sample1);
-$stmt = sqlsrv_query($conn, $query, $params);
-if ($stmt === false) {
-    die(print_r(sqlsrv_errors(), true));
-}
-
-$query = 'SELECT TOP 2 * FROM #TESTTABLE';
-$stmt = sqlsrv_query($conn, $query, array(), array("Scrollable"=>SQLSRV_CURSOR_CLIENT_BUFFERED));
-if (!$stmt) {
-    echo "Statement could not be prepared.\n";
-    die(print_r(sqlsrv_errors(), true));
-}
```

Comment:
why ignore the stmt checking?

Reply:
AE\insertRow already checked $stmt, but I'll double check